[Qwen3-Next] MOE configs for A100-SXM4-80GB TP4 TP8

toulzx opened the pull request:
rerun with:
```bash
# vllm==0.11.0
# benchmark_moe.py refer to https://github.com/vllm-project/vllm/blob/v0.11.0/benchmarks/kernels/benchmark_moe.py
python benchmark_moe.py --model Qwen/Qwen3-Next-80B-A3B-Instruct -tp 4  --tune  --trust-remote-code
python benchmark_moe.py --model Qwen/Qwen3-Next-80B-A3B-Instruct -tp 8  --tune  --trust-remote-code
```

toulzx committed:
Qwen3-Next-80B-A3B-Instruct's MOE configs for A100-SXM4-80GB TP4
---
vllm 0.10.2
`uv run benchmark_moe.py --model Qwen/Qwen3-Next-80B-A3B-Instruct -tp 4  --tune  --trust-remote-code`
---
Signed-off-by: tou <57480529+toulzx@users.noreply.github.com>

jeejeelee commented:
Could you tune using the latest main branch?  On one hand, the tuned config will include Triton version information, and on the other hand, the tuning results might be different with the latest Triton version

toulzx committed:
Add Qwen3-Next-80B-A3B-Instruct's MOE configs for A100-SXM4-80GB TP8
---
vllm 0.11.0
`python3  benchmark_moe.py  --model Qwen/Qwen3-Next-80B-A3B-Instruct  -tp 8  --dtype  auto   --tune  --trust-remote-code`
---
Signed-off-by: tou <57480529+toulzx@users.noreply.github.com>

toulzx committed:
Update Qwen3-Next-80B-A3B-Instruct's MOE configs for A100-SXM4-80GB TP4
---
vllm 0.11.0
`python3  benchmark_moe.py  --model Qwen/Qwen3-Next-80B-A3B-Instruct  -tp 4  --dtype  auto   --tune  --trust-remote-code`
---
Signed-off-by: tou <57480529+toulzx@users.noreply.github.com>

toulzx commented:
@jeejeelee   qwen3-next tuned moe configs PR for A100 with vllm==0.11.0 is ready now :)
